### PR TITLE
NORALLY: revising queries and supporting policy code format

### PR DESCRIPTION
**Pull request by graju256**
Revising existing queries in favor of new GraphQL types
Removed unnecessary queries
Supporting policy code fields dynamically by introducing a properties section as part of graphman.configuration
 - _policyCodeFormat_ decides what is the field used for specifying the policy code for export operations

### CODE REVIEW

File: modules/graphql-query-builder.js [hunk 1]
```diff
@@ -123,3 +119,10 @@ function buildQuery(queryIdPrefix, queryIdSuffix) {
         throw "unrecognized query " + queryIdPrefix;
     }
 }
+
+function substitutePolicyCodeField(text, policyCodeFormat) {
+    if (!policyCodeFormat || policyCodeFormat === "xml") return text;
```
Comment (uppoju) [inline modules/graphql-query-builder.js:124]: is it == ?

File: queries/encass.gql [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
+    policyByName(name: $policyName) {
+        {{L7Policy:-policy,policyRevision,policyRevisions}}
         policy {
             xml
-            directDependencies @include (if: $includeDirectDependencies) {
-                {{PolicyDependency}}
-            }
             allDependencies @include (if: $includeAllDependencies) {
-                {{PolicyDependency}}
+                {{PolicyDependency:-policyFragments,ldaps,fips,fipUsers,fipGroups}}
```
Comment (uppoju) [inline queries/encass.gql:10]: why we are excluding ldaps and fips ?

File: modules/graphql-query-builder.js (hunk 1 above)
Comment (graju256) [reply modules/graphql-query-builder.js:124]: === is more specific way of comparison. == leads to unnecessary type conversions.

File: queries/encass.gql (hunk 2 above)
Comment (graju256) [reply queries/encass.gql:10]: as they are deprecated ones, we would like to avoid them wherever it is possible.

**Review by uppoju — APPROVED** [review]

**Pull request merged**